files/build_templates/docker_image_ctl.j2

```diff
@@ -108,6 +108,10 @@ function preStartAction()
     fi
 {%- elif docker_container_name == "snmp" %}
     $SONIC_DB_CLI STATE_DB HSET 'DEVICE_METADATA|localhost' chassis_serial_number $(decode-syseeprom -s)
+{%- elif docker_container_name == "dhcp_relay" %}
+    until [[ $($SONIC_DB_CLI APPL_DB EXISTS "PORT_TABLE:PortInitDone" | grep -c 1) -gt 0 ]]; do
```

Comment:
Could you add log here then we could know dhcp_relay pending on waiting interface done?